pyClient/test/test_contract_base_mixer.py

```diff
@@ -16,30 +16,30 @@
 # it is structured as follows,
 UNPACKED_PRIMARY_INPUTS = [
     0,  # rt
-    8,  # nf_0 = "...1 000"
-    9,  # nf_1 = "...1 001"
-    18,  # cm_0 = "...10 010"
-    19,  # cm_1 = "...10 011"
+    24,  # nf_0 = "...11 000"
```

Comment:
`...11 000` -> `0...011 000` (to make it explicit that the prefixes are all 0's and not arbitrary data)
Likewise for the other lines below

Reply:
Updated.

Reply:
Not sure the other lines where updated, would you mind doing so?

Reply:
Updated.

Reply:
Thanks